README.md

```diff
@@ -38,8 +38,9 @@ You can run every example with zero setup (no registration needed).
 | Use Neptune in HPO jobs | [![docs]](https://docs.neptune.ai/tutorials/hpo) | [![neptune]](https://app.neptune.ai/o/common/org/pytorch-integration/experiments?split=tbl&dash=Loss-vs-Accuracy-bf72be6c-d771-457f-8f51-30fef2bee3d5&viewId=97f35039-00d1-43ac-9422-3f0ee5b2b0df) | [![github]](how-to-guides/neptune-hpo/scripts/Neptune_hpo_single_run.py) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/neptune-hpo/notebooks/Neptune_hpo.ipynb) |
 | Use Neptune in pipelines | [![docs]](https://docs.neptune.ai/usage/pipelines) | [![neptune]](https://app.neptune.ai/o/common/org/pipelining-support/runs/details?viewId=standard-view&detailsTab=metadata&shortId=PIP-3) |[![github]](how-to-guides/sequential-pipelines/scripts)| |
 | Reproduce Neptune runs | [![docs]](https://docs.neptune.ai/tutorials/reproducing_runs/) | [![neptune]](https://app.neptune.ai/o/common/org/showroom/runs/table?viewId=9968dc7f-61ee-49c3-a9a5-9006f10c6714&detailsTab=metadata&shortId=SHOW-30719&type=run) | [![github]](how-to-guides/reproduce-run/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/reproduce-run/notebooks/reproduce_run.ipynb) |
-| Restart runs from checkpoint | [![docs]]() | [![neptune]](https://app.neptune.ai/o/common/org/showroom/runs/details?viewId=standard-view&detailsTab=metadata&shortId=SHOW-32776&type=run) | [![github]](how-to-guides/restart-run-from-checkpoint/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/restart-run-from-checkpoint/notebooks/neptune_save_restart_run_from_checkpoint.ipynb)
+| Restart runs from checkpoint | [![docs]](https://docs.neptune.ai/tutorials/restarting_from_checkpoint/) | [![neptune]](https://app.neptune.ai/o/common/org/showroom/runs/details?viewId=standard-view&detailsTab=metadata&shortId=SHOW-32776&type=run) | [![github]](how-to-guides/restart-run-from-checkpoint/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/restart-run-from-checkpoint/notebooks/neptune_save_restart_run_from_checkpoint.ipynb)
 | Use Neptune in distributed computing | [![docs]](https://docs.neptune.ai/usage/distributed_computing) | | [![github]](how-to-guides/ddp-training/scripts)| |
+| Track models end-to-end | [![docs]](https://docs.neptune.ai/tutorials/e2e_tracking/) | [![neptune]](https://app.neptune.ai/o/showcase/org/e2e-tracking/runs/table?viewId=9b71afba-648f-40b7-9c70-98dc99bebc66) | [![github]](how-to-guides/e2e-tracking/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/e2e-tracking/notebooks/e2e_tracking.ipynb)
```

Comment:
```
[![docs]](https://docs.neptune.ai/tutorials/e2e_tracking/)
```

If you include placeholder URLs, please indicate it more clearly :) I think it'll be slightly different

Reply:
```suggestion
| Track models end-to-end | [![docs]](https://docs.neptune.ai/tutorials/tracking_models_e2e/) | [![neptune]](https://app.neptune.ai/o/showcase/org/e2e-tracking/runs/table?viewId=9b71afba-648f-40b7-9c70-98dc99bebc66) | [![github]](how-to-guides/e2e-tracking/scripts) | [![colab]](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/e2e-tracking/notebooks/e2e_tracking.ipynb)
```